ts-node-plugin/index.js

```diff
@@ -0,0 +1,38 @@
+const {transform} = require("../dist");
+
+const ModuleKindCommonJS = 1;
```

Comment:
@cspotcode Having to reference TypeScript's internal enum values is probably the most awkward part of the plugin interface, though it seems very understandable in giving the most flexibility in mapping TS options to transpiler options. I'm assuming it's extremely unlikely that TypeScript would change these values?

Reply:
That's my understanding, yeah.  FWIW the swc plugin already duplicates some of those enums.  Not sure if it's helpful for ts-node to expose these lookups as part of the plugin API to make things easier on your end?

https://github.com/TypeStrong/ts-node/blob/97f9afd046b66a0fe05a7d76e7a32f94b872016f/src/transpilers/swc.ts#L81-L129

Reply:
Yep, I saw what the swc plugin was doing and copied that approach. Exposing named values (rather than just numbers) could be nice, though I could imagine that being a pain to maintain as TS adds more options and more enum values, and it sounds like I'd only be able to use whatever is defined in the minimum ts-node version that I support. So my impression is that the current approach makes the most sense.